openid-sharedsignals-framework-1_0.md

```diff
@@ -165,15 +165,15 @@ and Coordination (RISC) and the Continuous Access Evaluation Profile ({{CAEP}})
 
 This specification defines:
 
-* A profile for Security Events Tokens {{RFC8417}}
+* A Profile for Security Events Tokens {{RFC8417}}
 * Subject Principals
 * Subject Claims in SSF Events
 * Event Types
 * Event Properties
-* Configuration information and discovery method for Transmitters
+* Configuration Metadata and Discovery Method for Transmitters
```

Comment:
both terms Configuration Metadata and Configuration Information used. Updated to always use Configuration Metadata

Reply:
I feel all of these should be in sentence case, e.g. "Subject principals", "Event types", etc. As they're terms that aren't yet defined in the spec or aren't referring to specifically previously defined terms anywhere. We should capitalize references to specified terms - either in this spec or other specs, where the first such reference should have the link to the other spec.

Reply:
that works for me. i will update this list.